rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -611,17 +612,34 @@ void emu_settings::EnhanceRadioButton(QButtonGroup* button_group, emu_settings_t
 		return;
 	}
 
-	for (int i = 0; i < options.count(); i++)
+	for (int i = 0, unchecked = 1, def_pos = -1; i < options.count() + unchecked; i++)
```

Comment:
I'd like to move the default setter out of this loop. I found it rather confusing to read.
Something like:
```
bool found = false;
int def_pos = -1;

// loop () {}

if (!found && def_pos >= 0)
{
	button_group->button(def_pos)->setChecked(true);
}
```